core/ui/ViewTemplate.tid

```diff
@@ -6,6 +6,6 @@ tc-tiddler-frame tc-tiddler-view-frame $(missingTiddlerClass)$ $(shadowTiddlerCl
 \define folded-state()
 $:/state/folded/$(currentTiddler)$
 \end
-<$set name="storyTiddler" value=<<currentTiddler>>><$set name="tiddlerInfoState" value=<<qualify "$:/state/popup/tiddler-info">>><$tiddler tiddler=<<currentTiddler>>><div class=<<frame-classes>>><$list filter="[all[shadows+tiddlers]tag[$:/tags/ViewTemplate]!has[draft.of]]" variable="listItem"><$transclude tiddler=<<listItem>>/></$list>
+<$set name="storyTiddler" value=<<currentTiddler>>><$set name="tiddlerInfoState" value=<<qualify "$:/state/popup/tiddler-info">>><$tiddler tiddler=<<currentTiddler>>><div data-title=<<currentTiddler>> data-tags={{!!tags}} class=<<frame-classes>>><$list filter="[all[shadows+tiddlers]tag[$:/tags/ViewTemplate]!has[draft.of]]" variable="listItem"><$transclude tiddler=<<listItem>>/></$list>
```

Comment:
Hi @pmario `data-title=<<currentTiddler>>` isn't consistent with the `data-tiddler-title` that we now have in the core. Could you kindly refactor?

Reply:
Yea. ... I missed that one, or I did forget about it :)) ... Will fix it

Reply:
@Jermolene ... What do you think about my proposal to add a "class" field similar to the "color" field of tiddlers. .. See: https://groups.google.com/d/msg/tiddlywiki/nlXjxlHRLmw/vKIAiMdGBQAJ TLDR; section?

Reply:
The "class” field is a great idea, go for it, if you've time, thanks!